src/main/io/gps.c

```diff
@@ -1540,132 +1712,145 @@ static bool UBLOX_parse_gps(void)
     uint32_t i;
 
     *gpsPacketLogChar = LOG_IGNORED;
-
     switch (_msg_id) {
-    case MSG_POSLLH:
-        *gpsPacketLogChar = LOG_UBLOX_POSLLH;
-        //i2c_dataset.time                = _buffer.posllh.time;
-        gpsSol.llh.lon = _buffer.posllh.longitude;
-        gpsSol.llh.lat = _buffer.posllh.latitude;
-        gpsSol.llh.altCm = _buffer.posllh.altitudeMslMm / 10;  //alt in cm
-        gpsSetFixState(next_fix);
-        _new_position = true;
-        break;
-    case MSG_STATUS:
-        *gpsPacketLogChar = LOG_UBLOX_STATUS;
-        next_fix = (_buffer.status.fix_status & NAV_STATUS_FIX_VALID) && (_buffer.status.fix_type == FIX_3D);
-        if (!next_fix)
-            DISABLE_STATE(GPS_FIX);
-        break;
-    case MSG_DOP:
-        *gpsPacketLogChar = LOG_UBLOX_DOP;
-        gpsSol.dop.pdop = _buffer.dop.pdop;
-        gpsSol.dop.hdop = _buffer.dop.hdop;
-        gpsSol.dop.vdop = _buffer.dop.vdop;
-        break;
-    case MSG_SOL:
-        *gpsPacketLogChar = LOG_UBLOX_SOL;
-        next_fix = (_buffer.solution.fix_status & NAV_STATUS_FIX_VALID) && (_buffer.solution.fix_type == FIX_3D);
-        if (!next_fix)
-            DISABLE_STATE(GPS_FIX);
-        gpsSol.numSat = _buffer.solution.satellites;
+        case MSG_NAV_POSLLH:
```

Comment:
Please revert to original indentation. This makes PR impossible to review

Reply:
must have slipped in sorry, was not intentional